pkg/plugins/resources/gitlab/mergerequest/spec.go

```diff
@@ -84,4 +84,58 @@ type Spec struct {
 			"body" is useful to provide additional information when reviewing mergerequest, such as changelog url.
 	*/
 	Body string `yaml:",omitempty"`
+
+	/*
+		"assignees" contains the list of assignee to add to the merge request
```

Comment:
Nitpick but I find it better to use the comment syntax `//` over `/* */` because in the latter, our IDE add `\t` which kind of change the documentation visualisation such as:

https://www.updatecli.io/docs/plugins/resource/hcl/ 

vs 

https://www.updatecli.io/docs/plugins/resource/yaml/

Reply:
Noted, I will change it. Just was trying to do it in the same way as it already was